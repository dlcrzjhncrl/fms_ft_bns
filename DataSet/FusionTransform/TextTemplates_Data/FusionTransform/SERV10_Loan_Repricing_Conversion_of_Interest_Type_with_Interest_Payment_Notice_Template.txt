                ***** Rate Setting *****
	
	
Description: <Currency> Loan Repricing for the Deal <Deal_Name>.

Effective: <Loan_EffectiveDate> 

Borrower <Borrower_ShortName> in Facility <Facility_Name> has the following loans repricing: 

                                                           Current      
                                                           Reprice      
Description                                 Global Amount  Date         
----------------------------------------------------------------------------------------
 <PricingOption> Loan                               <RequestedAmount_1>    <Loan_EffectiveDate>  




Borrower <Borrower_ShortName> in Facility <Facility_Name> will make the following interest payments: 

Description                                 Global Amount
--------------------------------------------------------------------------------
 <PricingOption>                                         <RequestedAmount_2> 





Borrower <Borrower_ShortName> in Facility <Facility_Name> will have the following new loans: 

                                                                      
Pricing                                                               
Option                                    Global Amount               
----------------------------------------------------------------------------------------
 <NewPricingOption>                              <RequestedAmount_1>               




 


Borrower <Borrower_ShortName> in Facility <Facility_Name> rates have been set as follows:


                                                                    Next         
Pricing                                     Global                  Reprice      
Option                                      Amount   All In Rate    Date         
----------------------------------------------------------------------------------------
 <NewPricingOption>                         <RequestedAmount_1>      <AllInRate_1>                 
	 <NewPricingOption>: <BaseRate_1>
	 Spread: <SpreadRate_1>
	      Projected Interest: <CycleDue_1>     
	 Projected  Interest Due Date: <ProjectedIntDueDate_1>








Deal ISIN: <DEAL_ISIN>
Deal CUSIP: <DEAL_CUSIP>
Facility: <Facility_Name> 

Facility ISIN: <FACILITY_ISIN>
Facility CUSIP: <FACILITY_CUSIP>



We will charge your account for an amount of <Currency> <RequestedAmount_2> on the effective date.

For: <Borrower_ShortName>
<Remittance_Description>


Account No:  
<RI_Description>
Name:  
<RI_AcctName>1
Routing Number:  
 


Reference: <Deal_Name>, 






|<comment>|