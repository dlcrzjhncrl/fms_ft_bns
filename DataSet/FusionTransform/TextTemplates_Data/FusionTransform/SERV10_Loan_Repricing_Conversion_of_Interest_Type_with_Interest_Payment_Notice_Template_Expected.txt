                ***** Rate Setting *****
	
	
Description: USD Loan Repricing for the Deal DEAL_NAME SC07144.

Effective: 18-Feb-2015 

Borrower AGENCYSBLCBORROWER44117 in Facility CRED01TERM39124 has the following loans repricing: 

                                                           Current      
                                                           Reprice      
Description                                 Global Amount  Date         
----------------------------------------------------------------------------------------
 Libor Option Loan                               10,000.00    18-Feb-2015  




Borrower AGENCYSBLCBORROWER44117 in Facility CRED01TERM39124 will make the following interest payments: 

Description                                 Global Amount
--------------------------------------------------------------------------------
 Libor Option                                         2.83 





Borrower AGENCYSBLCBORROWER44117 in Facility CRED01TERM39124 will have the following new loans: 

                                                                      
Pricing                                                               
Option                                    Global Amount               
----------------------------------------------------------------------------------------
 Fixed Rate Option                              10,000.00               




 


Borrower AGENCYSBLCBORROWER44117 in Facility CRED01TERM39124 rates have been set as follows:


                                                                    Next         
Pricing                                     Global                  Reprice      
Option                                      Amount   All In Rate    Date         
----------------------------------------------------------------------------------------
 Fixed Rate Option                         10,000.00      3.000000%                 
	 Fixed Rate Option: 1.000000%
	 Spread: 2.000000%
	      Projected Interest: 5.83     
	 Projected  Interest Due Date: 25-Feb-2015








Deal ISIN: 000402625-01
Deal CUSIP: 037833100
Facility: CRED01TERM39124 

Facility ISIN: 000402625-01
Facility CUSIP: 037833100



We will charge your account for an amount of USD 2.83 on the effective date.

For: AGENCYSBLCBORROWER44117
Demand Deposit Account


Account No:  
DDA1
Name:  
DDA1
Routing Number:  
 


Reference: DEAL_NAME SC07144, 






|<comment>|